Production/crab/config/2018/DY_embedded_Prompt_14.txt

```diff
@@ -0,0 +1,5 @@
+globalTag=102X_dataRun2_Prompt_v14 sampleType=Emb_18ABC runSVfit=False runKinFit=False storeLHEinfo=False saveGenParticleInfo=False isEmbedded=True
```

Comment:
sampleType should be Emb_18D, right? (please, cross-check)

Reply:
where can I cross-check that? the twiki of embbeded samples? ( I don't see anything like that)

Reply:
it is in the code: https://github.com/hh-italian-group/h-tautau/blob/prod_v5/Production/python/sampleConfig.py#L21 And I forgot that Maria Teresa already implemented GetGlobalTag  as a function of the sampleType (https://github.com/hh-italian-group/h-tautau/blob/prod_v5/Production/python/sampleConfig.py#L65), so no globalTag parameter should be specified in any of the configs... sorry, I forgot that.